db/migrate/20160122143711_add_scheduling_status_to_system_services.rb

```diff
@@ -0,0 +1,5 @@
+class AddSchedulingStatusToSystemServices < ActiveRecord::Migration
+  def change
+    add_column :system_services, :scheduling_status, :string
+  end
+end
```

Comment:
@petrblaho Can you please pull this migration out into a separate PR so that it can be merged independent of the rest of the changes in this PR?  We are approaching the deadline for schema changes for the darga branch, and I don't want the rest of the requested fixes in this PR to hold that up.